File: CPP/vcsDemo/DataModel/HandUpNotify.cpp
#include "HandUpNotify.h"

void HandUpNotify::Load(QJsonObject obj)
{
	if (!obj.isEmpty())
	{
		JsonUtil::GetValue(obj, "account_id", _account_id);
		JsonUtil::GetValue(obj, "account_name", _account_name);
		JsonUtil::GetValue(obj, "hus", _hus);
		JsonUtil::GetValue(obj, "result", _result);
		JsonUtil::GetValue(obj, "room_id", _room_id);
	}
}

File: CSharp/vcsSdk/Models/HandUpNotify.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace vcsSdk.Models
{
    public class HandUpNotify
    {
        [JsonProperty("account_id")]
        public string AccountId { get; set; }
        [JsonProperty("account_ame")]
        public string AccountName { get; set; }
        [JsonProperty("hus")]
        public int Hus { get; set; }
        [JsonProperty("result")]
        public int Result { get; set; }
        [JsonProperty("room_id")]
        public string RoomId { get; set; }
    }
}
